java/gradle/ice.gradle

```diff
@@ -9,18 +9,23 @@ buildscript {
         }
     }
     dependencies {
-        classpath group: "gradle.plugin.com.zeroc.gradle.ice-builder", name: 'slice', version: "1.5.0"
         classpath "com.diffplug.spotless:spotless-plugin-gradle:6.25.0"
     }
 }
 
-apply plugin: com.zeroc.gradle.icebuilder.slice.SlicePlugin
 apply plugin: com.diffplug.gradle.spotless.SpotlessPlugin
 
 slice {
-    cppConfiguration = project.cppConfiguration
-    cppPlatform = project.cppPlatform
-    iceHome = new File(project.ext.topSrcDir).getCanonicalPath()
+    includeSearchPath = files("${project.ext.topSrcDir}/slice")
```

Comment:
Should we instead build the slice-tools plugin first and then use it? 
As opposed to having this separate slice logic when building Ice Java?

Reply:
This currently rely on using

```
pluginManagement {
    includeBuild "../tools/gradle"
}
```

And there is no extra step for publishing the plug-in locally.

Reply:
Ok.